horreum-backend/src/main/resources/db/changeLog.xml

```diff
@@ -2360,5 +2360,36 @@
             CREATE POLICY dataset_update ON dataset FOR UPDATE USING (has_role2(owner, 'tester'));
         </sql>
     </changeSet>
+
+    <changeSet id="58" author="jwhiting">
+        <createTable tableName="transformationlog">
+            <column name="id" type="bigint">
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="level" type="integer">
+                <constraints nullable="false" />
+            </column>
+            <column name="timestamp" type="timestamp without time zone">
+                <constraints nullable="false" />
+            </column>
+            <column name="testid" type="integer" />
+            <column name="runid" type="integer" />
+            <column name="message" type="text">
+                <constraints nullable="false" />
+            </column>
+        </createTable>
+        <sql>
+            GRANT select, insert, delete ON TABLE transformationlog TO "${quarkus.datasource.username}";
+            ALTER TABLE transformationlog ENABLE ROW LEVEL SECURITY;
+            CREATE POLICY cl_all ON transformationlog FOR ALL
```

Comment:
See above, it would be useful to let role `horreum.system` log exceptions as well.

Reply:
ok